include/alpaka/rand/RandStl.hpp

```diff
@@ -68,6 +68,35 @@ namespace alpaka
                 public:
                     std::mt19937 m_State;
                 };
+
+                //#############################################################################
+                //! The standard library's random device based on the local entropy pool.
+                //!
+                //! Warning: the entropy pool on many devices degrates quickly and performance
+                //!          will drop significantly when this point occures.
+                class RandomDevice
+                {
+                public:
+
+                    //-----------------------------------------------------------------------------
+                    RandomDevice() = default;
+                    RandomDevice(RandomDevice&&) :
```

Comment:
Is a move constructor required for generators in alpaka? I guess so, since `std::random_device` is not movable and you provided this implementation. But then I think explicitly adding a `= default` move constructor to `MersenneTwister` and, potentially, others would make the code more clear.

Reply:
During init in the factory below it is needed. otherwise it won't compile since random_device is not movable